js/src/personal/riyuan/components/project/RiyuanProjectsPage.tsx

```diff
@@ -1,12 +1,10 @@
-import { Text, Title, Container } from '@mantine/core'
+import { Container, Title, Text } from '@mantine/core'
 import styles from '@/personal/riyuan/Riyuan.module.css'
 
-export function Project() {
+// This component renders the projects page
```

Comment:
nit: Should use the JSDoc format a bit for it: `/** Example Comment */`
Also probably want to be as concise as possible while also giving more context. 
Similar to commit message summaries, the comments should be written in an imperative mood.

Focus on the Why and not the What. 

// This component renders the projects page ->
/** Renders a Page that highlights different projects completed by me for visitors to get a sense of the work I've completed */